[TS-346] Added line graphs and comparison

tungleu opened the pull request:
The equity data can be viewed by day/month/year/total
![image](https://user-images.githubusercontent.com/30036024/160312973-aac505f2-97e6-4b43-b464-e74d722f48d9.png)

compare return on main dashboard: 
![image](https://user-images.githubusercontent.com/30036024/160313120-071a3ce1-eb1a-4a6e-b3f4-7cf1964ec384.png)

Mary's and Mac's are static data since we haven't implemented favorite accounts and we legit only have 2 alpaca accounts working among the apps. 
Other dashboard: 
![image](https://user-images.githubusercontent.com/30036024/160313157-d41b36f0-4445-4c03-9aaa-8fde8067ecd9.png)

other dashboard comparison (all the data are real): 
![image](https://user-images.githubusercontent.com/30036024/160313178-00c62dd0-00d9-4831-86ab-a1620c27a8b6.png)

https://user-images.githubusercontent.com/30036024/160317101-3c770374-3b4e-4a87-b5ac-e231d7be56e2.mp4

AlyaNaseer reviewed (changes requested):
Can you add more padding? It's a little squishy between the options and the graph.
![image](https://user-images.githubusercontent.com/46854444/160431362-e42b0c72-479f-4fa2-aeba-d5830f6083e4.png)

Also I saw you removed the options in the modal, we don't want to offer that to the user?

tungleu commented:
> Can you add more padding? It's a little squishy between the options and the graph. ![image](https://user-images.githubusercontent.com/46854444/160431362-e42b0c72-479f-4fa2-aeba-d5830f6083e4.png)
> 
> Also I saw you removed the options in the modal, we don't want to offer that to the user?
Fixed
![image](https://user-images.githubusercontent.com/30036024/160436543-67ba06e7-03ff-47f7-bf3d-09621c811ee0.png)
I moved the options (day/month/year/total) into the main graph instead of modals, that's the best i could do given the data we have 😅

bowe99 commented on cypress/integration/E2E_tests/ComparisonToolTests/comparisonTool.spec.js:
```diff
@@ -12,39 +12,39 @@ describe("On comparison page, can", () => {
-    cy.get("[data-cy=from]").should("have.value", "1900-01-01");
-  });
+    cy.get("[data-cy=legend]").should("have.length", 3);
+    // cy.get("[data-cy=radio-buttons]").should("have.length", 3);
+  });
+  // it("select option", () => {
+  //   cy.checkRadioButton('day');
+  // });
+  // it("select 'Custom'", () => {
+  //   cy.checkRadioButton('custom');
+  //   cy.get("[data-cy=to]").should("have.value", now);
+  //   cy.get("[data-cy=from]").should("have.value", now);
+  // });
+  // it("enter custom 'From' date", () => {
+  //   cy.setDate("from", "2022", "Feb", "10", "2022-02-10");
+  //   cy.update();
+  //   cy.get("[data-cy=to]").should("have.value", now);
+  // });
+  // it("cannot select a 'To' date that is before 'From'", () => {
+  //   cy.get("[data-cy=to]").click();
+  //   cy.get("[data-cy=date-picker-to]").contains("2022").click({ force: true });
+  //   cy.get("[data-cy=date-picker-to]").contains("Jan").should("be.disabled");
+  //   cy.get("[data-cy=date-picker-to]").contains("Feb").click({ force: true });
+  //   cy.get("[data-cy=date-picker-to]").contains("9").should("be.disabled");
+  //   cy.get("[data-cy=date-picker-to]").contains("10").click();
+  // });
+  // it("reset 'From' date and enter custom 'To' date", () => {
+  //   cy.setDate("to", "2022", "Mar", "6", "2022-03-06");
+  //   cy.get("[data-cy=from]").clear({ force: true });
+  //   cy.update();
+  //   cy.get("[data-cy=from]").should("have.value", "1900-01-01");
+  // });
```
Same here

AlyaNaseer commented on src/components/ComparisonTool/ComparisonModal.vue:
```diff
@@ -94,90 +98,7 @@
                     v-model="radios"
                     mandatory
                     dense
-                  >
```
I think Siobhan meant you could add a div here and add "None" so it isn't displayed, instead of deleting the code

AlyaNaseer commented on src/components/ComparisonTool/ComparisonModal.vue:
```diff
@@ -90,94 +94,96 @@
-                      offset-y
-                      min-width="auto"
-                      :nudge-left="260"
+                  <div class="demo_purpose">
```
```suggestion
                  <div style="display: None;">
```
Do it this way so you don't need the <style> at the bottom of your vue file

AlyaNaseer commented on src/components/ComparisonTool/ComparisonModal.vue:
```diff
@@ -267,3 +360,8 @@ export default {
 </script>
+<style>
+.demo_purpose {
+  display: None;
```
You can remove this and do my suggested change instead :)

AlyaNaseer reviewed (approved):
LGTM!
